registry/flutter_cocoon.test

```diff
@@ -0,0 +1,6 @@
+contact=flutter-infra@google.com
+fetch=git clone https://github.com/flutter/cocoon.git tests
+fetch=git -C tests checkout 129db63f675a470c7eee2056efe31733c30ffe7b
+update=.
+# Runs flutter analyze and flutter test
+test=./test_utilities/bin/flutter_test_runner.sh app_flutter
```

Comment:
Should this have an os qualifier?

Reply:
Yes, you'll have to provide a different script for windows, see here for an example: https://github.com/flutter/tests/blob/master/registry/flutter_packages.test

Reply:
Is it sufficient to just have Cocoon test on mac/linux? I don't there's much extra gained testing on windows, but I can create a windows script if you think it would be helpful.

Reply:
We currently require that tests need to run on all platforms: https://github.com/flutter/tests/blob/a558e872a5c7dc6b1b075d507d522734fdf95924/registry/template.test#L115

Flutter-owned packages should probably set a good example here and follow our own rules :)

Reply:
Added, but I had to bump the cirrus timeout from 60m to 75m. I noticed the last submitted test ran at 55 minutes on Cirrus. Is this expected?

Reply:
i thought i'd changed the rule so it didn't have to be all platforms. maybe i only changed the script and need to also change the docs.

Reply:
rather than bump the timeout, we should shard it.

Reply:
@Hixie is there an existing way to do this, or would support have to be added in the customer test script? https://github.com/flutter/flutter/blob/master/dev/customer_testing/run_tests.dart